src/main/scala/intent/sbt/Runner.scala

```diff
@@ -49,12 +49,12 @@ class SbtRunner(
    * Called by SBT with all the classes that match our fingerprint.
    *
    */
-  def tasks(potentialTasks: Array[TaskDef]): Array[Task] =
+  def tasks(potentialTasks: Array[TaskDef | Null] | Null): Array[Task | Null] =
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.

**Overridden java method returns array**
Overridden java method returns an array. Java arrays have type `Array[T | Null]`. Returned array must thereby be converted to have type `Array[T | Null]`.